src/frontend/handlers.go

```diff
@@ -189,6 +189,16 @@ func (fe *frontendServer) productHandler(w http.ResponseWriter, r *http.Request)
 		Price *pb.Money
 	}{p, price}
 
+	// Fetch packaging info (weight/dimensions) of the product
+	// The packaging service is an optional microservice you can run as part of a Google Cloud demo.
+	var packagingInfo *PackagingInfo = nil
+	if isPackagingServiceConfigured() {
+		packagingInfo, err = httpGetPackagingInfo(id)
+		if err != nil {
+			fmt.Println("Failed to obtain product's packaging info:", err)
+		}
+	}
```

Comment:
nit: if `isPackagingServiceConfigured` is never used except in conjunction with ` httpGetPackagingInfo`, what is a reason having it as a stand alone function? consider doing this check inside `httpGetPackagingInfo` and processing two errors: http vs not defined